resources/js/api/insurance.ts

```diff
@@ -0,0 +1,42 @@
+import type { ServiceResponse } from "./axios";
+import { api } from "./axios";
+
+export interface EligibilityResponseType {
+  benefit: "pharmacy" | "dme" | null;
+  is_eligible: boolean;
+  member_id: string;
+  payer: string;
+  state: string;
+}
+
+export interface EligibilityParamsRequestType {
+  firstName: string;
+  lastName: string;
+  dob: string;
+  memberId: string;
+}
+
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+export const checkBenefitsEligibilityQuery = {
+  mutation: async ({
+    firstName,
+    lastName,
+    dob,
+    memberId,
+  }: EligibilityParamsRequestType) => {
+    await delay(2000);
```

Comment:
we are probably gonna have to handle these things in the backend, and just have 1 request

Reply:
I talked to Rainier and he is developing for me to do 2 requests. We can have a huddle the three of us to recheck this if you have time